hydra-node/test/Hydra/Events/RotationSpec.hs

```diff
@@ -101,6 +104,45 @@ spec = parallel $ do
             events' <- getEvents (eventSource eventStore')
             let checkpoint' = foldl' mkAggregator s0 events'
             checkpoint `shouldBe` Checkpoint checkpoint'
+      it "a restarted and non-restarted node have consistent rotation" $ \testHydrate -> do
+        -- prepare inputs
+        now <- getCurrentTime
+        let contestationDeadline = toNominalDiffTime cperiod `addUTCTime` now
+        let closeInput = observationInput $ OnCloseTx testHeadId 0 contestationDeadline
+        let inputs = inputsToOpenHead ++ [closeInput]
+        let inputs1 = take 3 inputs
+        let inputs2 = drop 3 inputs
+        failAfter 1 $
+          do
```

Comment:
Nitpick: we usually put the `do` on the same line as can be also seen in this module.